package.json

```diff
@@ -45,6 +45,7 @@
     "@bull-board/express": "3.8.1",
     "@elastic/elasticsearch": "7.16.0",
     "@elastic/elasticsearch-mock": "0.3.1",
+    "@senecacdot/satellite": "1.17.0",
```

Comment:
@DukeManh with pinning in `pnpm` and your `release` change, do we continue to use `1.x`?

Reply:
We should have a PR for changing `1.17` to `1.x` if Renovate doesn't do that for us.

Also, I didn't see any Renovate PR today, looking at the Renovate logs, I don't think I got the config for scheduling right. I understand a bit more about it now.

```
DEBUG: Found timezone(branch="renovate/bull-board-express-3.x")
{
  "timezone": "America/Toronto"
}
DEBUG: Adjusting now for timezone(branch="renovate/bull-board-express-3.x")
DEBUG: Checking 1 schedule(s)(branch="renovate/bull-board-express-3.x")
DEBUG: Checking schedule "before 3am on Monday"(branch="renovate/bull-board-express-3.x")
{
  "parsedSchedule": {
  ...
  }
}
DEBUG: Package not scheduled(branch="renovate/bull-board-express-3.x")
DEBUG: Skipping branch creation as not within schedule
```
<img width="954" alt="image" src="https://user-images.githubusercontent.com/51073515/148832513-5b3a196f-8b64-4e75-aa68-f3620f998305.png">


Renovate runs check quite often, not really periodically, (from 16hours ago to 11 hours ago) and only if the check falls within the schedule window, PRs are created. We should increase the schedule window so Renovate has a chance to make PRs.

Reply:
@DukeManh I read up on your work on adding Renovate.

> We should have a PR for changing 1.17 to 1.x if Renovate doesn't do that for us.

Did you mean having a PR for changing from 1.x to 1.17 if Renovate doesn't do that for us. Because I looked at #2634, every dependency seems to be updated to pinned version

Reply:
@DukeManh I read up on your work adding Renovate

> We should have a PR for changing 1.17 to 1.x if Renovate doesn't do that for us.

Did you mean "changing from 1.x to 1.17" because I looked at #2634, every dependency seems to be updated to pinned versions

Reply:
Hey @DukeManh, how should I proceed from here. I supposed everything else is fine and we'll change satellite version if Renovate doesn't do it for us?

Reply:
Oh, this shouldn't block the progress. I'd say you can go with `1.17` and I will have another PR for updating all Satellite versions.

Reply:
As long as that gets filed.  If we say it but don't file it, it won't happen.  @TueeNguyen I'm fine for you to leave it to a follow-up, but please file an issue to do it.

Reply:
Duke just opened a PR for it!

Reply:
He's too good